springwolf-plugins/springwolf-spring-cloud-rabbit-binder-plugin/build.gradle

```diff
@@ -0,0 +1,145 @@
+plugins {
+    id 'java-library'
+    id 'maven'
+    id 'signing'
+    id 'maven-publish'
+
+    id 'org.springframework.boot' version '2.6.2'
+    id 'io.spring.dependency-management' version '1.0.6.RELEASE'
+}
+
+group 'io.github.springwolf'
+version '0.3.1'
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    api project(':springwolf-core')
+
+    implementation 'org.springframework:spring-web'
+    implementation 'org.springframework.cloud:spring-cloud-stream-binder-rabbit:3.2.4'
+
+    implementation 'com.google.guava:guava:27.0.1-jre'
+
+    compileOnly 'org.projectlombok:lombok:1.18.4'
+    annotationProcessor 'org.projectlombok:lombok:1.18.4'
+
+    testImplementation('org.springframework.boot:spring-boot-starter-test')
+
+    testCompile 'org.projectlombok:lombok:1.18.4'
+    testAnnotationProcessor 'org.projectlombok:lombok:1.18.4'
+
+    testCompile('org.assertj:assertj-core:3.11.1')
+}
+
+jar.enabled = true
+bootJar.enabled = false
+
+task javadocJar(type: Jar) {
+    classifier = 'javadoc'
+    from javadoc
+}
+
+task sourcesJar(type: Jar) {
+    classifier = 'sources'
+    from sourceSets.main.allSource
+}
+
+artifacts {
+    archives javadocJar, sourcesJar
+}
+
+signing {
+    sign configurations.archives
+}
+
+def pomConfig = {
+    licenses {
+        license {
+            name 'The Apache License, Version 2.0'
+            url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+        }
+    }
+
+    developers {
+        developer {
+            id 'stavshamir'
+            name 'Stav Shamir'
+            email 'shamir.stav@gmail.com'
+        }
+    }
+
+    scm {
+        url 'https://github.com/springwolf/springwolf-core'
+    }
+}
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            from components.java
+
+            artifact sourcesJar {
+                classifier "sources"
+            }
+
+            artifact javadocJar {
+                classifier "javadoc"
+            }
+
+            groupId project.group
+            artifactId 'springwolf-amqp'
```

Comment:
We should think of a name for the plugin springwolf-spring-cloud-rabbit-binder feels too long. Maybe springwolf-cloud-rabbit, or springwolf-cloud-rabbit-binder?

Reply:
I think springwolf-cloud-rabbit-binder will do.